README.md

```diff
@@ -1,3 +1,78 @@
-# Tracking ML
+# Tracking MLflow jobs on Openshift
 
-Demo of tracking machine learning metrics into mlflow running on k8s
+This is a demo application which includes templates that can be used to run mlflow jobs on Openshift.
+
+## Running on OpenShift
+This is a two stage process, because you need to create the image only once and then you can create multiple containers using the same image but with different parameters.
```

Comment:
I wouldnt even mention images and container. talk about `the code` and `experiment`.

Reply:
@durandom I tried to simplify this a little more, so if you could give it another read